packages/engine/format/runner_outbound_msg.fbs

```diff
@@ -28,11 +28,13 @@ struct TaskId {
 //    `task_id`        : the uuid for the task
 //    `metaversioning` : included when any batches have changed
 //                       (possibly since the last `StateSync`)
+//    `group_index`    : the index of the group of agents
```

Comment:
```suggestion
//    `group_index`    : If the task will be executed on a single group,
//                                the index of the group of agents.
//                                TODO: Otherwise this should be set to a null
//                                            value to indicate that the task will be
//                                            executed on all groups.
```

Reply:
Have updated it to something along those lines, let me know if you're happy with it